### PR TITLE
feat(react-wildcat): Pass server request / response into React as context

### PR DESCRIPTION
**Pull request by doctyper**

### CODE REVIEW

File: packages/react-wildcat-handoff/test/stubFixtures.js [hunk 1]
```diff
@@ -107,6 +107,13 @@ exports.requests = {
     }
 };
 
+exports.response = {
+    "access-control-allow-origin": "*",
+    "access-control-allow-methods": "GET,HEAD,PUT,POST,DELETE",
```
Comment (potench) [inline packages/react-wildcat-handoff/test/stubFixtures.js]: Do you need to support `OPTIONS`?

Comment (doctyper) [inline packages/react-wildcat-handoff/test/stubFixtures.js]: It's just a stub response. It does not impact the test suite results.

File: packages/react-wildcat-handoff/src/utils/serverContext.js [hunk 2]
```diff
@@ -5,21 +5,29 @@ const createReactClass = require("create-react-class");
 
 const RouterContext = Router.RouterContext;
 
-module.exports = function serverContext(request, headers, renderProps) {
+module.exports = function serverContext(
+    {request, response}, // eslint-disable-line react/prop-types
```
Comment (potench) [inline packages/react-wildcat-handoff/src/utils/serverContext.js]: This looks like a change in argument signature. Do you also need to change `simpleRender.js` to include the `response`?

Comment (doctyper) [inline packages/react-wildcat-handoff/src/utils/serverContext.js]: 👍

**Pull request merged**